playbooks/robusta_playbooks/image_pull_backoff_enricher.py

```diff
@@ -205,10 +202,12 @@ def investigate(
             error_message = self.get_kubelet_image_pull_error_from_event(
                 pod_event, container_status.image
             )
+            logging.info(f"for {pod_event} got message: {error_message}")
```

Comment:
Is it a leftover, or you really wanted to add this log in info level?
Might be a little spammy